pkg/controllers/dynakube/istio/conditions.go

```diff
@@ -0,0 +1,30 @@
+package istio
+
+import (
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func getConditionTypeName(component string) string {
+	return "IstioFor" + component
+}
+
+func setServiceEntryUpdatedConditionForComponent(conditions *[]metav1.Condition, component string) {
+	condition := metav1.Condition{
+		Type:    getConditionTypeName(component),
+		Status:  metav1.ConditionTrue,
+		Reason:  "IstioFor" + component + "Changed",
+		Message: "ServiceEntries and VirtualServices for " + component + " have been configured.",
+	}
+	_ = meta.SetStatusCondition(conditions, condition)
+}
+
+func setServiceEntryFailedConditionForComponent(conditions *[]metav1.Condition, component string, err error) {
+	condition := metav1.Condition{
+		Type:    getConditionTypeName(component),
+		Status:  metav1.ConditionFalse,
+		Reason:  "IstioFor" + component + "Failed",
+		Message: "Failed to configure Istio ServiceEntries and VirtualServices for " + component + " with error: " + err.Error(),
```

Comment:
@0sewa0 do we have limit lenght for message? if err is long ? nvm already found:
```
	// message is a human readable message indicating details about the transition.
	// This may be an empty string.
	// +required
	// +kubebuilder:validation:Required
	// +kubebuilder:validation:MaxLength=32768
	Message string `json:"message" protobuf:"bytes,6,opt,name=message"`
```